proto-lens-protoc/app/protoc-gen-haskell.hs

```diff
@@ -63,7 +65,10 @@ makeResponse dflags prog request = let
     header f = "{- This file was auto-generated from "
                 <> (f ^. #name)
                 <> " by the " <> pack prog <> " program. -}\n"
+    features = [CodeGeneratorResponse'FEATURE_PROTO3_OPTIONAL]
     in defMessage
+           & #supportedFeatures .~
+               (foldl (.|.) zeroBits $ fmap (toEnum . fromEnum) features)
```

Comment:
Could we simplify this to `supportedFeatures .~ toEnum (fromEnum FEATURE_PROTO3_OPTIONAL)` for now? We can add the `foldl` logic later once we have more than one feature to report.

Reply:
I would argue not to. Performance isn't an issue here. From a simplicity point of view, the `foldl` expression is certainly longer, but it does convey the message that this is an or-ed bit field without looking at the proto comments. So in my opinion it makes reading easier, since I as a reader would expect this to be multi-valued and surprised to not see a list. Also surprised to see the conversion from enum to int, which makes little sense for a single value but much for a bit field. The readers most likely to actually care about this code would be those adding another feature, in which case not having to come up with the `foldl` will save time.

Have I convinced you? If not, I will simplify.